.flake8

```diff
@@ -2,4 +2,4 @@
 max-line-length = 88
 extend-ignore =
   E203,
-exclude = build/*, env/*, venv/*, .nox, mogc_partnerships/migrations/*
+exclude = build/*, env/*, venv/*, .venv/*, deactivate/*, .nox, mogc_partnerships/migrations/*
```

Comment:
What is `deactivate`?

Reply:
I was wondering that too 😆 I have it in my local and looks to be added from pyvenv. Maybe I did this accidentally at some point without realizing it?